src/Components/Movies.js

```diff
@@ -0,0 +1,65 @@
+import axios from "axios";
+import { Component } from "react";
+
+class Movies extends Component {
+  state = {
+    movies: [],
+    title: "",
```

Comment:
It is better to have one piece of state keeping track of the entire movie and then you can access the information from there